docs/Theta/ThetaSetOpsCornerCases.md

```diff
@@ -85,7 +85,7 @@ The octal digit `ID = ((theta == 1.0) ? 4 : 0) | ((retainedEntries > 0) ? 2 : 0)
 | Shorthand Notation                | theta | retained entries |    empty   | Has Seen Data | ID | Comments                       |
 |:---------------------------------:|:-----:|:----------------:|:----------:|:-------------:|:--:|:------------------------------:|
 | New {1.0,0,T}                     |  1.0  |         0        |     T      |       F       |  5 | New Sketch, p=1.0 (default)    |
-| Exact {1.0,>0,F}                  |  1.0  |        >0        |     F      |       T       |  6 | Exact Mode                     |
+| Exact {1.0,>0,F}                  |  1.0  |        >0        |     F      |       T       |  7 | Exact Mode                     |
 | Estimation {<1.0,>0,F}            | <1.0  |        >0        |     F      |       T       |  2 | Estimation Mode                |
```

Comment:
This is not correct.  6 is the correct value, not 7.   Look at lines 79-81 of the raw text file.

The other typo fix in line 179 is fine.

Reply:
I'll fix the one remaining typo.  So I will close this. Thanks!

Reply:
My mistake, I was interpreting empty = false as 1 instead of 0 as per line 81.  Thanks for clarifying.